### DISCUSSION TITLE
Does Downgrading Change versions still work?

### DISCUSSION THREAD

**Discussion by gomes-web** [3kh0/ext-remover#1031] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Was wondering if Downgrading Change versions still works.

**Comment by helloIlovegimkit** [comment, marked as answer]
Yes it does though I'm pretty sure your chromebooks kernver which you can check by:

esc+refresh+power activate dev mode for old models (ofc blocked)

and check the kernver from there.

Or if your on drawcia or a newer model and get the dark-mode screen then:

esc+refresh+power debug log scroll down a bit or just find kernver.

If your kernver ends in a 2 or lower you can downgrade if it ends in a 3 you can't

**Reply by gomes-web** [reply]
tyy

**Reply by helloIlovegimkit** [reply]
You're welcome

**Reply by Yeetusmans2048** [reply]
kernver 0 or 1: any version
kernver 2: 112+
kernver 3: 120+
